### ISSUE TITLE
Add 429 response

### ISSUE THREAD

**Issue by ajrice6713** [jmulford-bandwidth/Bandwidth-Emulator#1]
It would be really useful if this emulated a configurable rate limit and returned 429 errors accordingly.

Please consider adding support for this uWu 🥺 
👉 👈

**Comment by ckoegel** [comment]
yeah the lack of support on this repo is honestly really disappointing.
